package-res/resources/web/pentaho/util/arg.js

```diff
@@ -20,23 +20,73 @@ define([
 
   var A_slice = Array.prototype.slice;
 
+  /**
+   * @name pentaho.util.arg
+   * @class
+   * @amd pentaho/util/arg
+   *
+   * @classDesc Util methods for keyword args.
+   *
+   * The AMD module returns the arg utils.
+   *
```

Comment:
Sorry, I read the line 30 " The AMD module returns the arg utils." and assumed you forgot to add the @amd tag.
When you run "ant doc-js", doesn't that line add duplicate information?